### ISSUE TITLE
v11.5.2 broke asset processing (AssetProcessingTimeout)

### ISSUE THREAD

**Issue by restfulhead** [contentful/contentful-management.js#2075]
We wanted to upgrade to the latest version, but noticed that processing assets after the upload consistently fails with the following error:  `AssetProcessingTimeout - "Asset is taking longer then expected to process."`

I went through the release notes and noticed that there was a change regarding uploads mentioned in `v11.5.2`. So I upgraded to the previous version `11.5.1` and everything works as usual. So something in `11.5.2` must be the cause of the AssetProcessingTimeout error.

Here's the code that I'm using to upload and then process assets:

```
const assetResult = await this.client.asset
        .createFromFiles(
          {
            spaceId: this.cmsSpace,
            environmentId: this.cmsEnv,
          },
          {
            fields: {
              title: {
                [language]: fileName,
              },
              description: {
                [language]: '',
              },
              file: {
                [language]: {
                  contentType: fileType,
                  fileName,
                  file: data,
                },
              },
            },
          }
        )

      const assetProcessed = await this.client.asset.processForAllLocales(
              {
                spaceId: this.cmsSpace,
                environmentId: this.cmsEnv,
              },
              asset
            )
```

**Comment by theochp** [comment]
I have been stuck on this for a few days, thank you for creating the issue.

**Comment by t-col** [comment]
hi @restfulhead thank you reporting. We reverted the problematic commit and released that fix in `v11.5.8`. We will reintroduce the asset processing changes from the original commit at a later time.

**Comment by jsdalton** [comment]
@restfulhead @theochp just wanted to give you both a heads up that we have shipped the API change again. further investigation revealed the original issue you both experienced was a backend issue and not related to the the API change.

if you do notice this issue (or something similar) please do feel free to open a new issue or comment here.

thanks for the original report and sorry about the difficulties here.
